src/content/docs/style-guide/writing-docs/writer-workflow/tech-writer-workflow.mdx

```diff
@@ -172,4 +172,20 @@ To create a branch on the docs-website repo:
 3. Click on **New Branch**
 4. You will be prompted to name your new branch. Descriptive names are best. It's a great way to quickly clue people in to what your work is all about. For example, if you are working on What’s New pages, you might name the branch **Whats-new-updates**.
 
-When you create a new branch, don't forget to add the Jira ticket's key (DOC-1234) to the branch name and the PR title. 
\ No newline at end of file
+When you create a new branch, don't forget to add the Jira ticket's key (DOC-1234) to the branch name and the PR title. 
+
+## Reviewing quickstart descriptions
+
+External vendors write quickstart descriptions. When one of these are ready for a review, they open an issue in our repo with a link out to the draft. These are written with SEO in mind, so they will appear differnt than most of our other docs and will not fully follow our style guide. 
```

Comment:
@bradleycamacho, 

* Verb agreement suggestion: "When one of these _is_ ready for a review..."
* No need for "out" in link out.
* Adverb: "so they will appear differently than..."